spec/lib/miq_automation_engine/miq_ae_service_model_spec.rb

```diff
@@ -136,4 +136,25 @@ module MiqAeServiceModelSpec
       end.to raise_error(NoMethodError)
     end
   end
+
+  context 'taggable' do
```

Comment:
No, I do not see the point in this for a few reasons:

1. Automate does not care if a specific model is taggable or not, that logic belongs in the backend.
1. There are so many models that support tagging that this list would be insanely long.
1. All we are really trying to do is verify the methods in the base automate service model work correctly with the active record models.  Since it is implemented at the base level I do not see the need to test if for every sub-classed model.  I believe we already have tests to cover it.